src/lib/components/ViewBoxControls.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import ToolboxButton from "$lib/components/ToolboxButton.svelte";
+  export let zooming: (zoomFactor: number) => void;
+</script>
+
+<div class="zoomButtons">
+  <ToolboxButton
+    text=" - "
+    callback={() => {
+      zooming(1.1);
+    }}
+  />
+  <ToolboxButton
+    text="Fullscreen"
+    callback={async () => {
+      if (!document.fullscreenElement) {
+        await document.documentElement.requestFullscreen();
+      } else {
+        await document.exitFullscreen();
+      }
+    }}
+  />
+  <ToolboxButton
+    text=" + "
+    callback={() => {
+      zooming(0.9);
+    }}
+  />
+</div>
+
+<style>
+  .zoomButtons {
+    position: absolute;
+    bottom: 5px;
+    right: 5px;
+  }
+</style>
```

Comment:
this should go with the other buttons css, hence why you have applied an id to the component yes? (src/lib/components/Network.svelte L28)

Reply:
Preferably we should just have a stylesheet right?

Reply:
No where ever possible we should try to maintain a concitent code base

Reply:
Do you mean that, because we've started using "local stylesheets" we shouldn't migrate to a global style sheet, which generally speaking is considered standard practice?

Reply:
I mean that the style rigt here shouldn't go her but in the parent file